Gemfile

```diff
@@ -22,10 +21,10 @@ def manageiq_plugin(plugin_name)
   end
 end
 
-manageiq_plugin "manageiq-schema"
+gem "manageiq-schema",         :git => "https://github.com/jrafanie/manageiq-schema.git",         :branch => "try_rails_5_2"
```

Comment:
Will need to drop this before MERGING

Reply:
We should start thinking about doing this if we are going to merge today.

Reply:
This is done now... just waiting for cross-repo tests to pass